docs/source/notebooks/LOv_rewriting_rules.ipynb

```diff
@@ -698,7 +698,7 @@
     }
    ],
    "source": [
-    "a=pcvl.Circuit.generic_interferometer(4, lambda idx:pcvl.Circuit(2)//pcvl.PS(phi=random.random())//pcvl.BS(theta=random.random()), depth=8, shape=\"rectangle\")\n",
+    "a=pcvl.Circuit.GenericInterferometer(4, lambda idx:pcvl.Circuit(2)//pcvl.PS(phi=random.random())//pcvl.BS(theta=random.random()), depth=8, shape=pcvl.InterferometerShape.RECTANGLE)\n",
```

Comment:
It should be `pcvl.GenericInterferometer`.
`Circuit` mistakenly stayed in the call.